trlx/trainer/accelerate_ppo_trainer.py

```diff
@@ -445,7 +445,7 @@ def make_experience(self, num_rollouts: int = 1024, iter_count: int = 0):  # noq
             sample_outputs = sample_outputs.cpu()
             values = values.cpu()[:, :-1]
 
-            ends = start + attention_mask[:, start:].sum(1)
+            ends = start + attention_mask[:, start:].sum(1) + 1
```

Comment:
Nit: Maybe add a comment saying something like "add 1 to account for appended eos  token" since the decoding is a few blocks up